guides/get_started_faster_with_forge.textile

```diff
@@ -71,116 +48,113 @@ h3. Setup Arquillian
 
-bc(output).. Which property do you want to set?
+bc(command). $ arquillian-container-configuration --container arquillian-wildfly-managed --container-option
 
-  1 - [Configuration{type='java.lang.String', name='jbossHome', description='The JBoss configuration to start.', defaultValue='$JBOSS_HOME'}]
-  ...
- ? Choose an option by typing the number of the selection: 1
- ? What value do you want to set? /home/ike/applications/jboss-as-7.1.1.Final
-Wrote /home/ike/projects/arquillian-demo/src/test/resources/arquillian.xml
+bc(output).  javaHome                 jbossHome                managementPort           outputToConsole          startupTimeoutInSeconds
+             javaVmArguments          managementAddress        modulePath               serverConfig
 
-p. Here's the resulting arquillian.xml file (formatted for clarity):
+Using above command, you can override any default property of your container configuration by setting it's appropriate value in arquilliain.xml using forge.
 
-div(filename). src/test/resources/arquillian.xml
+Note: In this example we are not going to override any default configuration property.
 
-bc(prettify). <arquillian xmlns="http://jboss.org/schema/arquillian"
-    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:schemaLocation="
-        http://jboss.org/schema/arquillian
-        http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
-    <container qualifier="JBOSS_AS_MANAGED_7.X">
-        <configuration>
-            <property name="jbossHome">/home/ike/applications/jboss-as-7.1.1.Final</property>
-        </configuration>
-    </container>
-</arquillian>
+p. Here's the resulting arquillian.xml file (formatted for clarity):
 
-In order to use this container configuration, you either have to:
+div(filename). src/test/resources/arquillian.xml
 
-* Add the attribute @default="true"@ to the @<container>@ element in arquillian.xml
-* Write the qualifier @JBOSS_AS_MANAGED_7.X@ as the only contents of the file src/test/resources/arquillian.launch
+bc(prettify). <?xml version="1.0" encoding="UTF-8" standalone="no"?>
+              <arquillian xmlns="http://jboss.org/schema/arquillian" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://jboss.org/schema/arquillian http://jboss.org/schema/arquillian/arquillian_1_0.xsd">
+                <container default="true" qualifier="arquillian-wildfly-managed">
+                  <configuration>
+                    <property name="chameleonTarget">${chameleon.target}</property>
+                  </configuration>
+                </container>
+              </arquillian>
 
 That's it! Arquillian is configured and ready to run tests inside JBoss AS 7.
 
 h3. Write a Test
 
 Now let's write a test. Once again, Forge can help you get started quickly.
 
-bc(command). $ arquillian create-test --class org.arquillian.example.MySimpleBean.java
+bc(command). $ arquillian-create-test --targets org.arquillian.tutorial.beans.MySimpleBean
```

Comment:
While migrating from forge 1.x to 2.x, if you create bean using forge command `cdi-new-bean --named MySimpleBean --scoped DEPENDENT` then it will automatically create class by name `org.project-name.beans.MySimpleBean`, So to create a test we are just using this bean as target which is anyway autocomplete